### ISSUE TITLE
HACS integration not found

### ISSUE THREAD

**Issue by SpeijerTom** [mikey0000/HA-Luba#17]
Hey awesome people, thank you for taking the time to create this integration.
I would love to test this, I have a LUBA 2 3000 however, when i try to add it, as described, i can't find it

![image](https://github.com/mikey0000/HA-Luba/assets/170707733/e2b26c25-4dd8-4fb0-abec-ec3e1920506f)

I tried in HACS and in the HA regular integrations

best regards, 
Tom

**Comment by romeoechovictor** [comment]
Did you try reboot the HASS? I think I had something similar in the beginning, but I also remember linking my github account with HACS and I've the repository saved on that as well... Maybe then it appeared?  I'm also a noob here with all this too.  Good luck getting it working, it's working great for me :)

**Comment by SpeijerTom** [comment]
Thank you, it worked by adding https://github.com/mikey0000/HA-Luba to the HACS custom integration.

**Comment by SpeijerTom** [comment]
I've been able to add the integration, but when starting the configuration, the screen remains blank 
Tried restarting HA multiple times

![image](https://github.com/mikey0000/HA-Luba/assets/170707733/142d961f-5b35-49e7-b16f-0d1404c25470)

**Comment by mikey0000** [comment]
Do you have an active Bluetooth device that is near Luba?

**Comment by SpeijerTom** [comment]
Well I have enabled Bluetooth + Active scanning on one of my Shelly Gen 2 devices, which is approx 3 meters away from the Luba2, but (or I am doing something wrong, because it's my first time using the bluetooth proxy feature in HA) I still get the empty mammotion screen. Tried some youtube videos to understand if there is something i still have to do, but can't figure it out, sorry

**Comment by nijel** [comment]
Shelly Bluetooth proxy is not really reliable for some time, there is issue about crashing script in the Home Assistant repo.

**Comment by SpeijerTom** [comment]
Hey Nijel, thank you for the quick response.
Problem is that Luba is in the yard and the closest i currently have is that Shelly.
Anything else you advise on using?

**Comment by romeoechovictor** [comment]
This is what I bought for the proxy. Works fine.

3 pack ESP32 ESP-32S Development Board 2.4 GHz Dual Core WLAN WiFi + Bluetooth 2-In-1 Microcontroller ESP-WROOM-32 Chip CP2102 for Arduino https://amzn.eu/d/0j6l3L5A

**Comment by mikey0000** [comment]
Yeah shelly do not support active proxy, so you'll need to setup an esp32, making sure it's set to active.

**Comment by tknice** [comment]
Yeah, I'm in the same boat.  Not sure how to easily connect bluetooth until wifi is supported.  I have plenty of dongles, just not one that can reach from the server to the other side of the house.  Are you saying there is a wifi supported esp32 based dongle for bluetooth that I can plug in anywhere?

Thank you mikey for working on this!!  So much appreciated.

**Comment by Kansalis** [comment]
You can flash a generic ESP32 as a bluetooth proxy from here: https://esphome.io/projects/index.html

They are very low power but also don't seem to have much range. As a test for this project I taped one to the top of my Luba2 with a battery pack. It maintains the bluetooth connectivity to my Home Assistant by roaming my wifi mesh as it moves around the garden. 

![image](https://github.com/mikey0000/HA-Luba/assets/38373647/d3f18377-bdce-45c2-bfa1-91e46a37a424)

I didn't leave it outside like that but the ESP32 is still plugged into the pack and has ~25% battery after 4 full days. I'm thinking of 3d printing a holster for the ESP32 & a battery to clip onto the top of the camera housing on the mower. If it's easily removable to bring inside to charge it's not much hassle to clip on before mowing while the project develops. While on the charging base I've put a second ESP32 in the shed close enough to keep a signal while it's stationary.

**Comment by tknice** [comment]
Thanks for the idea Kansalis, and kudos to you for trying the harder options. :)  I'd need decent range to still be on wifi (I can see the camera is dropping in and out at the edge of range around 70 yards away).
